public class usingthrowinexception
{
     public static void main(String args[])
{
    try{
        Throw new exception("error");
    }
    catch (exception)
    {
        System.out.println(e.getmesseage());
    }
    }
}
